### ISSUE TITLE
Cygwin make - creating libr/syscall/d/*.sdb with empty permissions

### ISSUE THREAD

**Issue by XVilka** [radare/radare2#2303]
```
$ ./configure &> ../cygwin64_configure.log ; make &> ../cygwin64_make.log ; make w32dist
make windist WINBITS=w32
make[1]: вход в каталог «/cygdrive/d/Work/radare/radare2»
[ -n "w32" ] || exit 1
rm -rf radare2-w32-0.9.9-git w32dist
mkdir w32dist
for a in `find libr | grep -e dll$`; do cp $a w32dist ; done
cp: пропускается каталог «libr/bin/d/dll»
for a in `find binr | grep -e exe$`; do cp $a w32dist ; done
rm -f w32dist/plugin.dll
mkdir -p w32dist/www
cp -rf shlr/www/* w32dist/www
mkdir -p w32dist/radare2/0.9.9-git/magic
cp -f libr/magic/d/default/* w32dist/radare2/0.9.9-git/magic
mkdir -p w32dist/radare2/0.9.9-git/syscall
cp -f libr/syscall/d/*.sdb w32dist/radare2/0.9.9-git/syscall
cp: cannot open «libr/syscall/d/darwin-x86-32.sdb» for reading: Permission denied
cp: cannot open «libr/syscall/d/darwin-x86-64.sdb» for reading: Permission denied
cp: cannot open «libr/syscall/d/dos-x86-16.sdb» for reading: Permission denied
cp: cannot open «libr/syscall/d/freebsd-x86-32.sdb» for reading: Permission denied
cp: cannot open «libr/syscall/d/ios-arm-32.sdb» for reading: Permission denied
cp: cannot open «libr/syscall/d/ios-arm-64.sdb» for reading: Permission denied
cp: cannot open «libr/syscall/d/ios-x86-32.sdb» for reading: Permission denied
cp: cannot open «libr/syscall/d/linux-arm-32.sdb» for reading: Permission denied
cp: cannot open «libr/syscall/d/linux-mips-32.sdb» for reading: Permission denied
cp: cannot open «libr/syscall/d/linux-sparc-32.sdb» for reading: Permission denied
cp: cannot open «libr/syscall/d/linux-x86-32.sdb» for reading: Permission denied
cp: cannot open «libr/syscall/d/linux-x86-64.sdb» for reading: Permission denied
cp: cannot open «libr/syscall/d/netbsd-x86-32.sdb» for reading: Permission denied
cp: cannot open «libr/syscall/d/openbsd-x86-32.sdb» for reading: Permission denied
cp: cannot open «libr/syscall/d/openbsd-x86-64.sdb» for reading: Permission denied
cp: cannot open «libr/syscall/d/windows-x86-32.sdb» for reading: Permission denied
cp: cannot open «libr/syscall/d/windows-x86-64.sdb» for reading: Permission denied
Makefile:64: Cannot make «windist»
make[1]: *** [windist] Error 1
make[1]: exiting «/cygdrive/d/Work/radare/radare2»
Makefile:56: Cannot make  «w32dist»
make: *** [w32dist] Error 2
```
And here are the permissions:
```
$ ls -l libr/syscall/d/*.sdb
---------- 1 a.kochkov user  10782 апр  3 14:44 libr/syscall/d/darwin-x86-32.sdb
---------- 1 a.kochkov user 10782 апр  3 14:44 libr/syscall/d/darwin-x86-64.sdb
---------- 1 a.kochkov user 11821 апр  3 14:44 libr/syscall/d/dos-x86-16.sdb
---------- 1 a.kochkov user  7498 апр  3 14:44 libr/syscall/d/freebsd-x86-32.sdb
---------- 1 a.kochkov user  28791 апр  3 14:44 libr/syscall/d/ios-arm-32.sdb
---------- 1 a.kochkov user  28791 апр  3 14:44 libr/syscall/d/ios-arm-64.sdb
---------- 1 a.kochkov user  28791 апр  3 14:44 libr/syscall/d/ios-x86-32.sdb
---------- 1 a.kochkov user   5747 апр  3 14:44 libr/syscall/d/linux-arm-32.sdb
---------- 1 a.kochkov user  5502 апр  3 14:44 libr/syscall/d/linux-mips-32.sdb
---------- 1 a.kochkov user  4525 апр  3 14:44 libr/syscall/d/linux-sparc-32.sdb
---------- 1 a.kochkov user  16501 апр  3 14:44 libr/syscall/d/linux-x86-32.sdb
---------- 1 a.kochkov user  27070 апр  3 14:44 libr/syscall/d/linux-x86-64.sdb
---------- 1 a.kochkov user  5901 апр  3 14:44 libr/syscall/d/netbsd-x86-32.sdb
---------- 1 a.kochkov user 16492 апр  3 14:44 libr/syscall/d/openbsd-x86-32.sdb
---------- 1 a.kochkov user  16519 апр  3 14:44 libr/syscall/d/openbsd-x86-64.sdb
---------- 1 a.kochkov user 138784 апр  3 14:44 libr/syscall/d/windows-x86-32.sdb
---------- 1 a.kochkov user 138784 апр  3 14:44 libr/syscall/d/windows-x86-64.sdb
```

**Comment by radare** [comment]
Ok, i know the fix for this. Will push in a while

**Comment by XVilka** [comment]
Fixed in 52741016d8c9a0fddeecb1089fb32419f46e01b3
Thanks!
